doc/tutorial.txt

```diff
@@ -310,11 +310,15 @@ events are simple action that will be performed by the thread or on the
 thread. They have to be listed by execution order.
 
 * run : Integer. Emulate the execution of a load. The duration is defined in
-usec but the run event will effectively run a number of time a loop that waste
-cpu cycles. When the run event is executed, the duration is transformed in a
-number of loop thanks to the ns per loop value (see calibration). This way of
-working enables to emulate a load with a duration that will vary with the
-frequency or the compute capacity of the CPU.
+ns but the run event will effectively run a number of time a loop that waste
```

Comment:
It's actually usec.

Reply:
is it? i have a calibration report of pLoad=45ns. setting 'run' to 10000 gives me a 'perf' column of 222, so that means 222 runs * 45ns / run = 9990ns, hence i  assumed it is in ns, not us - ?

Reply:
The event run is specified in us. The pload uses ns for precision

Reply:
222 runs of 45ns are not 10000us but 10000ns, though - which seems to suggest that the parameter is in ns. apologies if this is really obvious but... what am i missing?

Reply:
look at the code : 
```
	/*
	 * Run for the remainig exec (if any).
	 */
	load_count = (exec * 1000)/p_load;
	waste_cpu_cycles(load_count);

```
exec is the duration in usec which is then multiply by 1000 to becomes ns before being divided by p_load

but we return exec / p_load number of loop instead of (exec * 1000)/p_load

I can't remember exactly but don't we discuss that soem time ago Juri ? I think it was because of some overflow poblems that we return exec / p_load instead of (exec * 1000)/p_load

Reply:
ahh. i should've paid more attention. i just looked at the beginning, where it said
```
194         perf = exec / p_load; 
```

 thanks! so, perf in the log file is actually in 1000 pLoad units. correct? i think that needs adding to the documentation...

Reply:
yes. I agree

Reply:
updated the doc again and hopefully got the units right, this time...